Projects/Claims/Sources/Views/SubmitClaim/Entrypoints/SelectClaimEntrypoint.swift

```diff
@@ -0,0 +1,142 @@
+import Presentation
+import SwiftUI
+import TagKit
+import hAnalytics
+import hCore
+import hCoreUI
+
+public struct SelectClaimEntrypoint: View {
+    @PresentableStore var store: SubmitClaimStore
+    @State private var height = CGFloat.zero
+    @State private var tmpHeight = CGFloat.zero
+    @State var selectedClaimType: String? = nil
+
+    public init(
+        entrypointGroupId: String
```

Comment:
Switch this to optional
from showClaimEntrypointsOld send nil
from showClaimEntrypointsNew send value
We are handling hAnalytics in claims journey. Instead having it in more places

Also, we can make property let entrypointGroupId: String? which will be set on init
So instead of using hAnalyticsExperiment.claimsTriaging we can really on entrypointGroupId

Reply:
Fixed this